### ISSUE TITLE
Web is not serving

### ISSUE THREAD

**Issue by wahedzaman** [jameszah/ESP32-CAM-Video-Recorder-junior#24]
I uploaded the binary using the one click installer.  it was successfully uploaded and ran. I can see the video being recorded on sd however no way I can access the web interfaces. I tried all 80, 8080, 81, 82 ports but nothing is shown. I checked the log and the my router device list. It successfully connected to my router though. Any help will be appreciated. 

Regards

**Comment by jameszah** [comment]
Have a look at the serial monitor -- it should look like this below.
You should see a message of connected or failed, and the current time sync if the internet is working.   Then you should see those messages when you try to access the web server in the camera.  

Maybe some kind of router port blocking?  Or maybe just bad wifi - make sure there are no wires or metal (including a breadboad) near the antenna of the esp32. The antenna is the other side from the sd card.  You could try it in ap mode so there is no router issue - just your computer direct to the esp32.

![image](https://user-images.githubusercontent.com/36938190/192078790-7bc0a653-725b-4c29-94c9-5b609e9b0ccd.png)

**Comment by gitalexa** [comment]
i have similar issue

it is connecting to my accesspoint, but there is no webserver. 

<img width="861" alt="image" src="https://github.com/user-attachments/assets/a4e87525-d53c-4205-bd77-d6bf6b741fb4">

**Comment by jameszah** [comment]
can you ping it?
does it show up on your router?

Are you spelling out the entire ip: http://192.168.178.190/

Sometime desklens.local or http://desklens.local/ works.

It seems to getting the time from the internet.
When you attempt a connection to the server you should get the "http index, core 1" line above.

**Comment by gitalexa** [comment]
no, i can't ping it. It is not on my router (was a other device)

**Comment by gitalexa** [comment]
i removed almost all serial.prints and now i can reach the webserver. I don't know whether it is a coincindence or serial.print is consuming to much cpu-power...
